### ISSUE TITLE
Image refresh for rhel-8-0

### ISSUE THREAD

**Issue by cockpituous** [cockpit-project/cockpit#12110]
Image refresh for rhel-8-0
 * [ ] FAIL: image-refresh rhel-8-0

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-9.
Log: https://209.132.184.41:8493/logs/image-refresh-12110-20190619-204242/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-ci-srv-05.
Log: https://209.132.184.41:8493/logs/image-refresh-12110-20190619-205050/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-12110-20190619-205050/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-ci-srv-03.
Log: https://209.132.184.41:8493/logs/image-refresh-12110-20190620-130404/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-12110-20190620-130404/

**Comment by marusak** [comment]
```
DEBUG: No match for argument: golang-bin
DEBUG: No match for argument: golang-src
```

**Comment by marusak** [comment]
This is affected by https://github.com/cockpit-project/cockpit/pull/12055 as `bots/images/scripts/lib/build-deps.sh` was introduced. This script is not able to distinguish opsys and therefore macros on https://github.com/cockpit-project/cockpit/blob/master/tools/cockpit.spec#L801 for the `go_scl_prefix` do not work. I am afraid that this is going to need a different approach.

There is utility `rpmspec` which can do all kind of tricks with specfiles.
`rpmspec -q --buildrequires <specfile>` lists build requires directly. (`<specfile>` is cockpit.spec where `npm-version` has been replaced (`sed 's/%{npm-version:.*}/0/' tools/cockpit.spec`).

Also it is worth mentioning that this golang stuff is only for kubernetes subpackage. rpmspec needs to be called with `build_kubernetes` macro defined to parse this as well (or maybe not, if we don't want to build kubernetes).

Is using of rpmspec acceptable? If not, we need to make `build-deps.sh` much smarter.

**Comment by marusak** [comment]
Fixed in #12130
